frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -223,9 +217,36 @@ const getDefaultCustomAnalysisSettings = () => {
   return customVals;
 };
 
-const getArraysForWells = () => {
+const getDefaultAnalysisSettingsForWell = () => {
+  return {
+    windowedAnalysisBounds: {
```

Comment:
This is a global setting, there shouldn't be a copy for each well

Reply:
this is for one well, not all wells, and this is how it was getting passed to `waveformGraph`

Reply:
but you're right, I was able to remove it, at one point I thought I needed it at initialization because `customAnalysisSettingsForWell` couldn't be set until `selectedWell` and `customAnalysisSettings` are set correctly which can't be done until the waveform data has been received since we can no longer assume number of wells or that `A1` exists or is the first well